src/Sources/CdmChangeFeedSource/CdmChangeFeedSource.cs

```diff
@@ -41,14 +41,17 @@ public class CdmChangeFeedSource : GraphStage<SourceShape<List<DataCell>>>, IPar
     private readonly string rootPath;
     private readonly TimeSpan schemaUpdateInterval;
     private readonly bool stopAfterBackfill;
+    private readonly int lookBackRange;
 
     private CdmChangeFeedSource(string rootPath,
         string entityName,
         IBlobStorageService blobStorage,
         bool isBackfilling,
         TimeSpan changeCaptureInterval,
         bool stopAfterBackfill,
-        TimeSpan schemaUpdateInterval)
+        TimeSpan schemaUpdateInterval,
+        int lookBackRange
```

Comment:
mb lookBackRangeSeconds?

Reply:
The name is the same as in SQL change tracking source, we can rename it later in both sources.